.gitignore

```diff
@@ -14,3 +14,5 @@ Session.vim
 bin/
 /gopath/
 /release/
+
+.idea/
```

Comment:
Don't add changes to .gitignore in this PR.

Reply:
Sure, this can be done in another PR